changelog/issue-7938.md

```diff
@@ -0,0 +1,5 @@
+audience: users
+level: patch
+reference: issue 7938
+---
+D2G: no longer pass Docker Worker env vars through to Generic Worker env vars, only supply them as flags to the `docker run` command.
```

Comment:
This doesn't look like it's actually passing the env vars, though?

Reply:
We were always passing in the `docker run` command https://github.com/taskcluster/taskcluster/blob/31ad80ee8a3605e089821cdfdf12c67db2ead1f2/tools/d2g/d2g.go#L670-L673

Reply:
Nvm, I see what you're saying. I forgot we removed the value of the env var to the field being passed in. Moving to draft.